src/configure/index.ts

```diff
@@ -92,11 +92,11 @@ const configureGitLab = async () => {
     execSync(`glab auth login`, { stdio: "inherit" });
     execSync(`glab variable set OPENAI_API_KEY ${apiKey}`);
     logger.info(
-      "Successfully added the OPENAI_API_KEY secret to your GitLab repository."
+      "Successfully added the OPENAI_API_KEY secret to your GitLab repository.\n Please make sure you have set up your Gitlab access token before using this tool. Refer to the README (Gitlab CI section) for information on how to do this."
     );
```

Comment:
I thought it might be nice to add a reminder here about setting it up. Since if you run configure,and you assume that it wil set up everything for you, you  might not go check the readme. 
This makes sure that they are reminded that they need to do some manual intervention, before they go and try to run it and then it break and then only do they know to fix it. 

But I am happy for someone to challenge this!